[tgui] build failure

# Host Environment
OS: Windows 10 Pro Version 21H1 build 19043.1081 with Windows Feature Experience Pack 120.2212.3530.0
Compiler: Microsoft Visual Studio Community 2019 Version 16.10.2
# To Reproduce
Steps to reproduce the behavior:
``````````
./vcpkg install "tgui[core,tool]" --triplet x64-windows
``````````
# Failure logs
``````````
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:146 (message):
    Command failed: D:/vcpkg/downloads/tools/cmake-3.20.2-windows/cmake-3.20.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/vcpkg/buildtrees/tgui/x64-windows-dbg
    See logs for more information:
      D:\vcpkg\buildtrees\tgui\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:104 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:44 (vcpkg_build_cmake)
  ports/tgui/portfile.cmake:37 (vcpkg_install_cmake)
  scripts/ports.cmake:139 (include)


Error: Building package tgui:x64-windows failed with: BUILD_FAILED
``````````
[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/6729405/install-x64-windows-dbg-out.log)
# Additional context
The error is LNK2019: unresolved external symbol WinMain referenced in function "int __cdecl invoke_main(void)" in gui-builder.exe.

--- a/ports/tgui/vcpkg.json
+++ b/ports/tgui/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "tgui",
   "version-date": "2021-04-19",
-  "port-version": 1,
+  "port-version": 2,
   "description": "TGUI is an easy to use, cross-platform, C++ GUI for SFML.",
   "homepage": "https://tgui.eu",
   "default-features": [
@@ -35,7 +35,16 @@
       ]
     },
     "tool": {
-      "description": "Build GUI builder"
+      "description": "Build GUI builder",
+      "dependencies": [
+        {
+          "name": "tgui",
+          "features": [
+            "sdl2",
+            "sfml"
+          ]
+        }
+      ]
     }
   }
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3539,8 +3539,8 @@
     "libsigcpp-3": {
       "baseline": "3.0.3",
       "port-version": 1
-    }, 
-	"libsmb2": {
+    },
+    "libsmb2": {
       "baseline": "2021-04-29",
       "port-version": 0
     },
@@ -6150,7 +6150,7 @@
     },
     "tgui": {
       "baseline": "2021-04-19",
-      "port-version": 1
+      "port-version": 2
     },
     "theia": {
       "baseline": "0.8",

--- a/versions/t-/tgui.json
+++ b/versions/t-/tgui.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0fd517c4e738da62e959b791a6cdd952e7bd8345",
+      "version-date": "2021-04-19",
+      "port-version": 2
+    },
+    {
       "git-tree": "354b5135bda4bcef9c5e7cbfeaff3d457b336e8c",
       "version-date": "2021-04-19",
       "port-version": 1

